.github/AL-Go-Settings.json

```diff
@@ -1,76 +1,77 @@
-{
-    "type":  "PTE",

```

Comment:
consider keeping the same formatting

Reply:
It's a format mismatch will AL-Go. We'll find a solution at some point.

For better review experience, you can turn on the "Hide whitespace" setting:
![image](https://github.com/microsoft/BCApps/assets/43066499/9cfc2904-0323-48f9-b21c-02bf96fee644)